gms/factories/src/main/java/com/linkedin/ml/factory/MLModelDAOFactory.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.ml.factory;
```

Comment:
Note that you'll need to change the namespace if https://github.com/linkedin/datahub/pull/1916 gets in first :)

Reply:
#1916 is in now.